draft-ietf-scitt-architecture.md

```diff
@@ -367,43 +367,52 @@ This section describes at a high level, the three main roles and associated proc
+
+Transparency Services MUST reject any attempt to register Signed Statements until at least one Registration Policy is registered.
+
+Registration Policies MAY be empty, in which case the Transparency Service only promises to enforce the Mandatory Registration Checks.
 
-Registration Policies refer to the checks that are performed before a Signed Statement is registered to an Append-only Log, and a corresponding Receipt becomes available.
+This specification leaves implementation, encoding and documentation of Registration Policies to the operator of the Transparency Service.
+
+#### Mandatory Registration Checks
+
+In addition to enforcing the rules contained in any registered Registration Policy, Transparency Services MUST, at a minimum, perform the following checks before registering a Signed Statement:
+* Authenticate the Issuer of the Signed Statement
 
-As a minimum, a Transparency Service MUST authenticate the Issuer of Signed Statements, which requires a trust anchor in the form of an already registered Signed Statement including key material (see {{ts-initialization}}).
-As defined in {{RFC6024}}, "A trust anchor represents an authoritative entity via a public key and associated data.
+To authenticate the Issuer of the Signed Statement the Transparency Service MUST refer to a trust anchor as defined in {{RFC6024}}: "A trust anchor represents an authoritative entity via a public key and associated data.
 The public key is used to verify digital signatures, and the associated data is used to constrain the types of information for which the trust anchor is authoritative."
-Typical representations of a trust anchor include certificates or raw public keys.
 
-The `x5t` and `kid` Claims in the protected header of Signed Statements can be used as hints for discovering trust anchors.
-Before a Registration Policy is used to decide if a Signed Statement is registered, the policy MUST be registered.
-Before a Signed Statement is registered, the trust anchor used to verify it MUST be registered (e.g., via a registered Registration Policy).
-In order to register a trust anchor, the trust anchor MUST be converted to a Signed Statement with a matching content type Claim.
-During initialization of a Transparency Service, the first Signed Statements registered will be for a trust anchor that is not validated by any Registration Policy.
+Typical representations of a trust anchor include certificates or raw public keys. The `x5t` and `kid` Claims in the protected header of Signed Statements can be used as hints for discovering trust anchors.
 
-Transparency Services MUST specify their supported signature algorithms in their Registration Policies.
+Before a Signed Statement is registered, the trust anchor used to verify its Issuer MUST be registered with the Transparency Service. To enable this, Transparency Services MUST support at least one of these methods:
```

Comment:
Per [this comment](https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/201/files#r1488113904), I'd suggest removing lines 405-408